src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
If i move all this properties to * selector it will break construction

Reply:
I meant only `box-sizing`)